mlflow/ml_package_versions.py

```diff
@@ -249,6 +249,15 @@
             "maximum": "0.0.286"
         }
     },
+    "promptflow": {
+        "package_info": {
+            "pip_release": "promptflow"
+        },
+        "models": {
+            "minimum": "1.1.0",
+            "maximum": "2.0.0"
```

Comment:
any breaking changes between 1.1 and 2.0?

Reply:
looks like 2.0 is not released yet: https://pypi.org/project/promptflow/#history

Reply:
is it ok if we don't add the maximum? seems all the package have both minimum and maximum

Reply:
updated.